### ISSUE TITLE
The conditional check 'default_exists.stat.exists && main_template_enable' failed

### ISSUE THREAD

**Issue by snikiforov4** [nginxinc/ansible-role-nginx#20]
```
FAILED! => {"failed": true, "msg": "The conditional check 'default_exists.stat.exists && main_template_enable' failed. The error was: template error while templating string: unexpected char u'&' at 33. String: {% if default_exists.stat.exists && main_template_enable %} True {% else %} False {% endif %}

The error appears to have been in '__skipped__/roles/nginxinc.nginx/tasks/conf/template-config.yml': line 8, column 3, but may be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: \"(Setup: All NGINX) Backup NGINX Default Configuration File\"
  ^ here
This one looks easy to fix.  It seems that there is a value started\nwith a quote, and the YAML parser is expecting to see the line ended with the same kind of quote.  For instance:

    when: \"ok\" in result.stdout

Could be written as:

   when: '\"ok\" in result.stdout'

Or equivalently:

   when: \"'ok' in result.stdout\"
"}
```

Hi there! It seems that `&&` is not allowed. And would be better to use `and` instead.
Version of ansible is 2.4.4.0

**Comment by alessfg** [comment]
Good catch @snikiforov4 - I'll fix it asap.

**Comment by alessfg** [comment]
Fixed in #21 😄
